### PR TITLE
Common props

### PR DESCRIPTION
**Pull request by Fi1osof**

### CODE REVIEW

File: src/Modal.jsx [hunk 1]
```diff
@@ -88,15 +88,29 @@ class Modal extends React.Component {
...
-    <div>
+  render = () => {
+    
+    const {
+      wrapperProps,
+      className,
+      visible,
+      onClickBackdrop,
+      onOK,
+      onCancel,
```
Comment (nbudin) [inline src/Modal.jsx]: `onOK` and `onCancel` only apply to the `ConfirmModal` component, so they shouldn't be excluded here.  Perhaps a similar thing in `ConfirmModal` to do that would be good though.

File: src/Modal.jsx [hunk 2]
```diff
@@ -88,15 +88,29 @@ class Modal extends React.Component {
     return null;
   }
 
-  render = () => (
-    <div>
+  render = () => {
+    
+    const {
+      wrapperProps,
+      className,
```
Comment (nbudin) [inline src/Modal.jsx]: Please add `wrapperProps` and `className` to the `static propTypes` above.

File: src/Modal.jsx [hunk 3]
```diff
@@ -88,15 +88,29 @@ class Modal extends React.Component {
...
+      {...wrapperProps}
+    >
       <div
-        className={classNames('modal', 'fade', { show: this.state.visible })}
+        className={classNames('modal', 'fade', { show: this.state.visible }, className)}
         style={{ display: ((this.state.visible || this.state.transitioning) ? 'block' : 'none') }}
         role="dialog"
         aria-hidden={!this.state.visible}
         tabIndex="-1"
         onClick={this.props.onClickBackdrop}
```
Comment (nbudin) [inline src/Modal.jsx]: Now that we're explicitly grabbing `onClickBackdrop` into a const, let's use it here.

File: src/Modal.jsx [hunk 4]
```diff
@@ -88,15 +88,29 @@ class Modal extends React.Component {
...
+      wrapperProps,
+      className,
+      visible,
+      onClickBackdrop,
+      onOK,
+      onCancel,
+      ...other
+    } = this.props;
+
+    return <div
```
Comment (nbudin) [inline src/Modal.jsx]: Stylistic note: please wrap this whole JSX block in parentheses to make eslint happy?

**Comment by Fi1osof** [comment]
@nbudin is this OK? https://github.com/nbudin/react-bootstrap4-modal/pull/5/commits/165413674e3d0238c0c2fa3ef8a364a2994fa219

**Comment by nbudin** [comment]
@Fi1osof: much better, thanks!  I'll merge and do a little eslint cleanup and then push out a release.

**Comment by Fi1osof** [comment]
Thank!

**Comment by nbudin** [comment]
Version 1.2.0 is out, with this and some additional similar features for other parts of the modal and the ConfirmModal component.

**Pull request merged**